js/node-manager/src/client/components/Layout.tsx

```diff
@@ -316,9 +318,16 @@ export const Layout: React.FC<LayoutProps & ClassName> = ({
       </div>
       <div className="flex flex-col flex-grow overflow-hidden">
         <div className="flex items-center flex-shrink-0 h-16 px-7 border-b border-gray-300 z-10">
-          <h1 className="text-lg font-medium" id="Layout_Title">
-            {title}
-          </h1>
+          <div className="flex flex-col">
```

Comment:
[Non-major] Does changing between page make the text go up and down?
If so, then add `justify-start py-2` here